R/summaries.R

```diff
@@ -258,11 +261,22 @@ setMethod("countDominantTaxa", signature = c(x = "SummarizedExperiment"),
                      call. = FALSE)
             }
         }
+        # name check
+        if(!is.null(name)) {
+          if(is.character(name)==FALSE) {
+            stop("'name' must be a character value", call. = FALSE)
+          }
+        }
+        
+        # digits for rounding
+        args <- list(...)
+        digits <- args$digits
```

Comment:
THis line can be removed if you modify line
`
.tally_col_data(data, group, digits, colname, name)`

to 
`
.tally_col_data(data, group, digits, colname, name, ...)`

AND line

`.tally_col_data <- function(data, group, digits, colname, name){`

to
`
.tally_col_data <- function(data, group, digits = NULL, colname, name, ...){`

Your solution works, but for consistency (we have used this latter approach in other functions)